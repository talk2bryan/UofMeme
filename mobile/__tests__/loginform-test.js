import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import "isomorphic-fetch";

import LoginForm from '../components/Login/LoginForm';



it('renders correctly', () => {
  const tree = renderer.create(
    <LoginForm />
    ).toJSON();
  expect(tree).toMatchSnapshot();
});