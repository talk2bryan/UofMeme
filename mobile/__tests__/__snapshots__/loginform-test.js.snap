// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "white",
      "flex": 1,
      "marginBottom": 10,
      "padding": 20,
    }
  }
>
  <View
    keyboardVerticalOffset={0}
    onLayout={[Function]}
    style={
      Array [
        Object {
          "backgroundColor": "white",
          "flex": 1,
          "marginBottom": 10,
          "padding": 20,
        },
        Object {
          "paddingBottom": 0,
        },
      ]
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "justifyContent": "center",
        }
      }
    >
      <Image
        source={1}
        style={
          Object {
            "height": 100,
            "width": 100,
          }
        }
      />
      <Text
        accessible={true}
        allowFontScaling={true}
        ellipsizeMode="tail"
        style={
          Object {
            "color": "black",
            "fontSize": 20,
            "fontWeight": "bold",
            "marginBottom": 10,
            "marginTop": 10,
            "textAlign": "center",
            "width": 200,
          }
        }
      >
        Welcome to UofMeme
      </Text>
    </View>
    <View
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <TextInput
        allowFontScaling={true}
        autoCapitalize="none"
        autoCorrect={false}
        editable={true}
        keyboardType="email-address"
        onChangeText={[Function]}
        onSubmitEditing={[Function]}
        placeholder="email@myumanitoba.ca"
        placeholderTextColor="white"
        returnKeyType="next"
        style={
          Object {
            "backgroundColor": "gray",
            "color": "white",
            "height": 40,
            "marginBottom": 20,
            "paddingHorizontal": 10,
          }
        }
        value={undefined}
      />
      <TextInput
        allowFontScaling={true}
        editable={true}
        onChangeText={[Function]}
        placeholder="Password"
        placeholderTextColor="white"
        returnKeyType="go"
        secureTextEntry={true}
        style={
          Object {
            "backgroundColor": "gray",
            "color": "white",
            "height": 40,
            "marginBottom": 20,
            "paddingHorizontal": 10,
          }
        }
        value={null}
      />
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        collapsable={undefined}
        hasTVPreferredFocus={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        nativeID={undefined}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "backgroundColor": "darkgrey",
            "marginBottom": 20,
            "opacity": 1,
            "paddingVertical": 15,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "white",
              "fontWeight": "700",
              "textAlign": "center",
            }
          }
        >
          LOGIN
        </Text>
      </View>
      <View
        accessibilityComponentType={undefined}
        accessibilityLabel={undefined}
        accessibilityTraits={undefined}
        accessible={true}
        collapsable={undefined}
        hasTVPreferredFocus={undefined}
        hitSlop={undefined}
        isTVSelectable={true}
        nativeID={undefined}
        onLayout={undefined}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "backgroundColor": "#337ab7",
            "marginBottom": 20,
            "opacity": 1,
            "paddingVertical": 15,
          }
        }
        testID={undefined}
        tvParallaxProperties={undefined}
      >
        <Text
          accessible={true}
          allowFontScaling={true}
          ellipsizeMode="tail"
          style={
            Object {
              "color": "white",
              "fontWeight": "700",
              "textAlign": "center",
            }
          }
        >
          Not a meme-ber? Sign up now!
        </Text>
      </View>
    </View>
  </View>
</View>
`;
